jobbergate-composed/docker-compose.yml

```diff
@@ -57,6 +57,7 @@ services:
       - ../jobbergate-api/jobbergate_api/:/app/jobbergate_api/
       - ../jobbergate-api/dev_tools/:/app/dev_tools/
       - ../jobbergate-api/tests/:/app/tests/
+      - ../jobbergate-api/:/app/
```

Comment:
We need to take the file `coverage.xml` out of the container. @dusktreader, do you agree with this as an alternative to solve the issue?

Reply:
not asked to me but... I agree. Indeed, I do that in other projects across the company

Reply:
I'm not a big fan of volume mounting the whole project folder into the container. When the running app creates files in this folder, they are owned by root and can cause pain. If we are going to do this kind of mount, there's no point in mounting the source code, test folder, and dev_tools separately.

Reply:
That is a good point. Do you have a suggestion to copy just `coverage.xml` back from the container?

Reply:
You could change the path so that it is in a different folder that you can more safely volume mount.

Reply:
I redirected the report to `tests/coverage.xml`, so there is no need for mounting the whole project. I used the same path on other components for consistency.

Addressed in [0efb40c](https://github.com/omnivector-solutions/jobbergate/pull/472/commits/0efb40c7cab3e5e71b4ad8429a7c22915a182ca9).